toml.md

```diff
@@ -109,6 +109,12 @@ may contain ASCII digits, dashes and underscores. Punctuation, spaces, arrows, b
 and private use characters are not allowed.  Note that bare keys are allowed to be
 composed of only ASCII digits, e.g. `1234`, but are always interpreted as strings.
 
+* From the ASCII characters, only A-Z, a-z, 0-9, _ and - are allowed
+* From the rest of the first 256 characters, only 0080-00BF, "×" (00D7) and "÷" (00F7) are disallowed
+* All of 0100-1FFF are allowed, except ";" (037E)
+* Characters 200C, 200D, 203F, 2040, 2070-218F, 2C00-2FEF are allowed
+* All characters from 3001, except surrogates (D800-D999), Private Use (E000-F8FF, F0000-100000) and process-internal use (FDD0-FDEF)
```

Comment:
That means something like "U+3001 and higher", right? It's not quite clear.

Reply:
I'll update

Reply:
done